common/changes/office-ui-fabric-react/pchandoria-contextualMenu-fix_2018-05-11-00-54.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Breaking BaseButton Types dependency from ContextualMenu class",
+      "type": "patch"
```

Comment:
I'm not certain this is a `patch`. What if people used to depend on props from `ContextualMenu`? @dzearing / @aditima - what's your take. When I chatted with @pgonzal we believe this a breaking change that we may need to hold off till 6?

Reply:
They can still use them because it's any. This won't be breaking.

Reply:
On second though `any` covers everything and the `ContextMenu` was itself pointing back to `IContextualMenuProps`